src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ConfiguredProjectImplicitActivationTracking.Instance.cs

```diff
@@ -41,6 +46,8 @@ public Task InitializeAsync()
 
             protected override Task InitializeCoreAsync(CancellationToken cancellationToken)
             {
+                _dataProgressTrackerRegistration = _dataProgressTrackerService.RegisterForIntelliSense(this, _project, nameof(ConfiguredProjectImplicitActivationTrackingInstance));
```

Comment:
Should the name of the data source include additional information about the project and/or configuration? I wonder if that might be helpful when looking at a dump file with multiple configurations where it might not be clear which is still pending.